sci-libs/geos/geos-3.6.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+inherit autotools eutils python-single-r1 python-utils-r1
+
+DESCRIPTION="Geometry engine library for Geographic Information Systems"
+HOMEPAGE="http://trac.osgeo.org/geos/"
+SRC_URI="http://download.osgeo.org/geos/${P}.tar.bz2"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="amd64 arm ppc ppc64 x86 ~x64-freebsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris"
+IUSE="doc php python ruby static-libs"
+
+RDEPEND="
+	php? ( >=dev-lang/php-5.3:* )
+	ruby? ( dev-lang/ruby:* )
+"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )
+	php? ( dev-lang/swig )
+	python? ( dev-lang/swig ${PYTHON_DEPS} )
```

Comment:
`python-single-r1` in combination with `DEPEND`-only `${PYTHON_DEPS}` makes no sense